src/plugins/data/server/index_patterns/fetcher/index_patterns_fetcher.ts

```diff
@@ -58,16 +58,19 @@ export class IndexPatternsFetcher {
     rollupIndex?: string;
   }): Promise<FieldDescriptor[]> {
     const { pattern, metaFields, fieldCapsOptions, type, rollupIndex } = options;
+    const patternList = Array.isArray(pattern) ? pattern : pattern.split(',');
+    const patternListActive = await this.validatePatternListActive(patternList);
     const fieldCapsResponse = await getFieldCapabilities(
       this.elasticsearchClient,
-      pattern,
+      patternListActive.length > 0 ? patternListActive : patternList,
```

Comment:
if none of the patterns are active, pass the original list to get an error:
<img width="1252" alt="Screen Shot 2021-02-04 at 7 32 20 AM" src="https://user-images.githubusercontent.com/6935300/106907825-e8779280-66bb-11eb-921a-04386edf09c1.png">

Reply:
Would be good to add this as a comment to the code. It definitely makes sense but its not obvious why this is being done.